docs/src/pages/style/links/LinkUserExperience.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-script-url */
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+
+const styles = theme => ({
+  link: {
+    margin: theme.spacing.unit,
+  },
+});
+
+// This resolves to nothing and doesn't affect browser history
+const dudUrl = 'javascript:;';
+
+function Links() {
+  return (
+    <Typography>
+      Click <Link href={dudUrl}>here</Link> to see our latest offers.
```

Comment:
This example ignores the accessibility advice in the following demo...

Reply:
Hahah 2 different people 😆. For reference: https://developers.google.com/web/tools/lighthouse/audits/descriptive-link-text.

Reply:
Yup, that pretty much describes my philosophy, which I think originated with Jakob Nielsen in the '90s...

Reply:
:P Should probably be 'Check out our **latest offers**'